go.mod

```diff
@@ -61,11 +61,18 @@ require (
 )
 
 retract (
-	v7.0.0+incompatible
-	v6.0.0+incompatible
-	v5.0.0+incompatible
-	v4.0.0+incompatible
-	v3.0.0+incompatible
-	v2.0.0+incompatible
-	v1.0.0
+	// Published v7 too early (before migrating to go modules)
+	v7.0.0
+	// Published v6 too early (before migrating to go modules)
+	v6.0.0
+	// Published v5 too early (before migrating to go modules)
+	v5.0.0
+	// Published v4 too early (before migrating to go modules)
+	v4.0.0
+	// Published v3 too early (before migrating to go modules)
+	v3.0.0
+	// Published v2 too early (before migrating to go modules)
+	v2.0.0
+	// Published v1 too early (before migrating to go modules)
+	[v1.0.0, v1.0.1]
```

Comment:
```
$ go list -m -versions github.com/u-root/u-root
github.com/u-root/u-root v0.1.0 v0.2.0 v0.3.0 v0.4.0 v0.5.0 v0.6.0 v0.7.0 v0.8.0 v1.0.0 v2.0.0+incompatible v3.0.0+incompatible v4.0.0+incompatible v5.0.0+incompatible v6.0.0+incompatible v7.0.0+incompatible
```

Couple things:

1. I don't see a v1.0.1
2. If we tag v7.0.1, after that retraction, our last version will be v0.8.0, which (a) does not exist in this repo anymore and (b) is probably super old. Should we also retract v0.1.0 through v0.8.0?

Reply:
Re 1) When merged, this commit must be tagged with v1.0.1 (not v7.0.1 as mentioned at the top) @walterchris has tested it.  This is why v1.0.1 also needs to be retracted.

Re 2) v7.0.1 tag is not required, see above. Why does v0.8.0 not exist? Retracting v.0.x.0 is up to you (and maybe something for another PR,  when this went well)

Reply:
Re 1) @walterchris has a v3.0.1 that also retracts v3.0.1 in his test repo: https://github.com/walterchris/goversion rather than a v1.0.1. I thought you had to choose a version higher than all versions you are retracting?

2) ignore the v7.0.1. If we retract all the major versions, what's left is v0.1.0 to v0.8.0. Those are old u-root versions. Then, when you pull github.com/u-root/u-root@latest, don't you get v0.8.0?

Reply:
Yes, we tried with publishing the retractions with v3.0.1 tag (including v3.0.1) but this did not work in the first place. Since v3.x.x isn't a valid version (not v3 folder/path/branch stuff)
The retractions become valid with introducing them with the v1.0.1 tag (including v1.0.1) Since this is the latest valid version.

> If we retract all the major versions, what's left is v0.1.0 to v0.8.0. Those are old u-root versions. Then, when you pull github.com/u-root/u-root@latest, don't you get v0.8.0?

That's true.